doc/api/crypto.md

```diff
@@ -656,6 +656,54 @@ assert.strictEqual(aliceSecret.toString('hex'), bobSecret.toString('hex'));
 // OK
 ```
 
+### ECDH.convertKey(key, curve[, inputEncoding][, outputEncoding][, format])
```

Comment:
Is this signature correct? According to this signature, all of the following calls would be valid:

```js
ECDH.convertKey(key, curve[, inputEncoding][, outputEncoding][, format])
ECDH.convertKey(key, curve[, inputEncoding][, outputEncoding])
ECDH.convertKey(key, curve[, inputEncoding])
ECDH.convertKey(key, curve)
ECDH.convertKey(key, curve[, inputEncoding][, format])
ECDH.convertKey(key, curve[, outputEncoding][, format])
ECDH.convertKey(key, curve[, format])
```

Reply:
@tniessen How would I structure the signature?

`key` and `curve` are required but the rest are optional.

but for example if `format` wants to be specified, `inputEncoding` and `outputEncoding` will also have to be specified. 
but you can also just specify `inputEncoding` and `outputEncoding` without specifying `format`

Would it be: `ECDH.convertKey(key, curve[, inputEncoding, outputEncoding, format])`